velox/connectors/hive/storage_adapters/abfs/RegisterAbfsFileSystem.cpp

```diff
@@ -34,12 +35,28 @@ std::shared_ptr<FileSystem> abfsFileSystemGenerator(
   });
   return filesystem;
 }
+
+std::unique_ptr<velox::dwio::common::FileSink> abfsWriteFileSinkGenerator(
+    const std::string& fileURI,
+    const velox::dwio::common::FileSink::Options& options) {
+  if (isAbfsFile(fileURI)) {
+    auto fileSystem =
+        filesystems::getFileSystem(fileURI, options.connectorProperties);
+    return std::make_unique<dwio::common::WriteFileSink>(
+        fileSystem->openFileForWrite(fileURI, {{}, options.pool, std::nullopt}),
```

Comment:
The second parameter is not used in `AbfsFileSystem.openFileForWrite`. Do we need to costruct it here?

Reply:
removed